client/src/components/Form/PasswordInput/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Input from '../FormInput';
+
+const PasswordInput = ({ verify, passwordToMatch, ...props }) => {
+  let error =
+    'Password should be of minimum 8 character long and contain atleast one uppercase, lowercase and special character';
+  let label = 'Password';
+  let id = 'password';
+  //TODO: uncomment below line after testing
```

Comment:
will be uncommented before final release since it will unnecessary force us to type password in the format